tests/playwright/src/setupFiles/setup-kind.ts

```diff
@@ -0,0 +1,33 @@
+/**********************************************************************
+ * Copyright (C) 2024-2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { isLinux, isMac, isWindows } from '../utility/platform';
+
+export function setupKind(): boolean[] {
+  const rootfulMode = process.env.ROOTFUL_MODE === 'true'; // undefined or "false" => false
+  const runKindTests = process.env.RUN_KIND_TESTS === 'true';
+  return [rootfulMode, runKindTests];
+}
+
+export function canRunKindTests(): boolean {
+  const [rootfulMode, runKindTests] = setupKind();
+  // to run the kind tests there are 2 options:
+  // 1-the OS is linux or mac + runKindTests == true
+  // 2-the OS is windows, the podman machine is rootful
+  return ((isLinux || isMac) && runKindTests) || (isWindows && rootfulMode);
```

Comment:
I would change the evaluation when to run kind tests.

One restriction is that it does not run on windows with rootless, right?
The other one is that we may have them running on linux or mac...
The last one is that if there is a RUN_KIND_TESTS==true, we want to run them.

So from my perspective: 
`return runKindTests || (isMac || isLinux) || (isWindows && rootfulMode)`.
This way, we have env. var which will trigger kind tests in any case (in cases when we want to run kind with preconfigured rootless - which is possible), then we want to run them on mac o on linux, and finally, we will run tests on windows rootfull.